thenewboston_node/business_logic/tests/test_file_blockchain/test_add_block.py

```diff
@@ -1,11 +1,185 @@
+from unittest.mock import patch
+
 import pytest
 
 from thenewboston_node.business_logic.blockchain.file_blockchain import FileBlockchain
 from thenewboston_node.business_logic.exceptions import ValidationError
 from thenewboston_node.business_logic.models.block import Block
+from thenewboston_node.business_logic.tests.mocks.storage_mock import StorageMock
 from thenewboston_node.core.utils.cryptography import KeyPair
 
 
+@pytest.fixture
+def blockchain(
+    forced_file_blockchain, initial_account_root_file, forced_mock_network, get_primary_validator_mock,
+    get_preferred_node_mock
+):
+    block_storage_mock = patch.object(forced_file_blockchain, 'block_storage', StorageMock())
+    arf_storage_mock = patch.object(forced_file_blockchain, 'account_root_files_storage', StorageMock())
+
+    with block_storage_mock, arf_storage_mock:
+        forced_file_blockchain.add_account_root_file(initial_account_root_file)
+        yield forced_file_blockchain
+
+
+class TestAddBlock:
+
+    def test_block_chunk_is_rotated(self, blockchain, user_account, signing_key):
+        file1 = '00000000000000000000-00000000000000000000-block-chunk.msgpack'
+        file2 = '00000000000000000001-00000000000000000001-block-chunk.msgpack'
+
+        with patch.object(blockchain, 'block_chunk_size', 1):
```

Comment:
please, use some other size like 3, to avoid false positives (one is somewhat a special number)